src/lua/httpc.lua

```diff
@@ -54,7 +64,10 @@ local http_new = function(opts)
     opts.max_total_connections = opts.max_total_connections or 0
 
     local curl = driver.new(opts.max_connections, opts.max_total_connections)
-    return setmetatable({ curl = curl, }, curl_mt )
+    return setmetatable({
+        curl = curl,
+        encoders = encoders,
+    }, curl_mt)
```

Comment:
So if we'll modify encoders of one particular http client instance, the encoders will be modified for all other instances? It seems, it shouldn't work this way.